config/routes.rb

```diff
@@ -5,11 +5,18 @@
   get '/jobs/new' => 'jobs#new'
   get '/pages' => 'pages#index'
   get '/home' => 'pages#show'
-  post '/users' => 'users#create'
   get '/signup' => 'users#new'
   post '/sessions' => 'sessions#create'
 
   get '/login' => 'sessions#new'
   post '/login' => 'sessions#create'
   delete '/logout' => 'sessions#destroy'
+
+  get '/users' => 'users#index'
+  post '/users' => 'users#create'
+  get '/users/:id' => 'users#show'
+  patch '/users/:id' => 'users#update'
+  delete '/users/:id' => 'users#destroy'
+
```

Comment:
to continue with my comment above about the `prefix` when creating custom routes you can this prefix as well. something like:
`get '/pages' => 'session#new', as: 'pages'`